packages/backend-core/tests/utilities/structures/tenants.ts

```diff
@@ -0,0 +1,5 @@
+import { newid } from "../../../src/newid"
+
+export function name() {
```

Comment:
```suggestion
export function id() {
```
Just as we already use `tenantId` everywhere, there is no mention of name outside of the account portal

Reply:
True 😅